### PR TITLE
Add read user coils and liquids

### PR DESCRIPTION
**Pull request by gbaranski**

### CODE REVIEW

**Comment by gbaranski** [comment]
My suggestion is to add EDIT button, and after click on it, it would redirect you to coil calculator page with filled inputs from saved on firebase coil.

**Comment by gbaranski** [comment]
Also i think great idea is to add some icons. Currently it looks like that. Thinking about button that reveals more info in modal.
Screen Shot 2020-06-17 at 03 42 08

**Comment by gbaranski** [comment]
Need to discuss with @stasbar if modal is good idea.

**Comment by stasbar** [comment]
This idea with edit button is great although it might be quite hard to do.

File: src/components/ItemView/CoilView.tsx [hunk 1]
```diff
@@ -31,32 +51,90 @@ class CoilView extends ItemView<Coil, ItemViewProps<Coil>, CoilViewState> {
...
-          )
-        }
-      >
-        <Card.Meta
-          avatar={<FirebaseImage type={ImageType.USER} id={item.author.uid} />}
-          description={<Typography.Text>{item.description}</Typography.Text>}
-        />
-        <this.Actions />
+      <>
+        <Modal
```
Comment (stasbar) [inline src/components/ItemView/CoilView.tsx:65]: Move to components directory

File: src/components/ItemView/CoilView.tsx [hunk 2]
```diff
@@ -1,23 +1,43 @@
...
+import { DownOutlined } from '@ant-design/icons';
 import { ItemView, ItemViewProps, ItemViewState } from './ItemView';
 import styles from './styles.less';
 
 interface CoilViewState extends ItemViewState {
   coilImageUrl: string;
+  modalVisible: any;
 }
 
+enum Tuple {
```
Comment (stasbar) [inline src/components/ItemView/CoilView.tsx:17]: Rename to `SetupsName`

File: src/components/ItemView/CoilView.tsx [hunk 3]
```diff
@@ -1,23 +1,43 @@
...
+  Single = 1,
+  Dual = 2,
+  Triple = 3,
+  Quad = 4,
+}
```
Comment (stasbar) [inline src/components/ItemView/CoilView.tsx:22]: VT supports up to octa coils

**Comment by stasbar** [comment]
It's always good idea to have icons, I will manage it for you

**Comment by stasbar** [comment]
You can start implementing liquids as well

**Pull request merged**